src/uk/gov/defra/ffc/DefraUtils.groovy

```diff
@@ -209,7 +237,19 @@ def notifySlackBuildFailure(exception, channel) {
   slackSend channel: channel,
             color: "#ff0000",
             message: msg.replace("  ", "")
+}
 
+def versionHasIncremented(currVers, newVers) {
```

Comment:
I can see why you've not added this to the readme as it's essentially a private function.  You can make a private function with the `private` keyword, however it seems there's a bug in Groovy and it treats it as public.  It still might be good to make our intention of if it's use clearer?  Or document it in the readme.

Reply:
I have added it now, but as it's internal it may be better to remove the documentation and make it private. What do you think?